config/features/config.go

```diff
@@ -75,6 +75,8 @@ type Flags struct {
 	CorrectlyInsertOrphanedAtts bool
 	CorrectlyPruneCanonicalAtts bool
 
+	EnableUseNativeState bool // EnableUseNativeState defines whether the beacon state will be represented as a pure Go struct or a Go struct that wraps a proto struct.
```

Comment:
```suggestion
	EnableNativeState bool // EnableNativeState defines whether the beacon state will be represented as a pure Go struct or a Go struct that wraps a proto struct.
```
This reads better, what do you think ?